### PR TITLE
Add step to get the supported instance types

### PR DESCRIPTION
**Pull request by yasun1**
[OCPQE-18903](https://issues.redhat.com//browse/OCPQE-18903)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-e2e-f3

File: ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh [hunk 1]
```diff
@@ -86,6 +86,7 @@ if [[ "$LOCAL_ZONE" == "true" ]]; then
     exit 1
   fi
 
+  MP_MACHINE_TYPE=$(tail -n 1 "${SHARED_DIR}/instance-types.txt")
```
Comment (radtriste) [inline ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh:89]: I would keep the configuration via env var for `MP_MACHINE_TYPE`, with default value from instance types

```suggestion
  MP_MACHINE_TYPE=${MP_MACHINE_TYPE:-$(tail -n 1 "${SHARED_DIR}/instance-types.txt")}
```

wdyt ?

**Comment by yasun1** [comment]
The changes are verified by the [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/47588/rehearse-47588-periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-e2e-f3/1745741177837064192).  

The next changes are only do OWNER file change.

File: ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh (hunk 1 above)
Comment (yasun1) [reply ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh:89]: It is special for the localzone. It depends on the zone in the step `aws-provision-localzone-subnet` so we can not specify the MP_MACHINE_TYPE randomly. If in the future, we have another way to prepare the localzone subnet, let use update the code with the new achievement.

**Pull request merged**